net/pfSense-pkg-pfBlockerNG-devel/files/usr/local/pkg/pfblockerng/pfblockerng_install.inc

```diff
@@ -167,8 +167,14 @@ stop_service('pfb_filter');
 update_status(" done.\nRenew Firewall filter executables...");
 unlink_if_exists('/usr/local/bin/php_pfb');
 link('/usr/local/bin/php', '/usr/local/bin/php_pfb');
-unlink_if_exists('/usr/local/sbin/clog_pfb');
-link('/usr/local/sbin/clog', '/usr/local/sbin/clog_pfb');
+if (file_exists('/usr/local/sbin/clog')) {
+                unlink_if_exists('/usr/local/sbin/clog_pfb');
```

Comment:
unlink should be out of if statement, we want clog_pfb link to be removed always

Reply:
also tail_pfb then ?